core/modules/filters/contains.js

```diff
@@ -0,0 +1,77 @@
+/*\
+title: $:/core/modules/filters/contains.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for comparing the contents of an input tiddlers field and the
+operand.
+
+This filter returns the input title iff the operand is a subset of the given
+field of the input.
+
+Example 1:
+
+MyList: 1 2
```

Comment:
what happens if I set: `MyList: 1 foo`

Reply:
Than it will return the tiddler because the tiddler also has foo in its field.